### ISSUE TITLE
Adding one marker redraws everything

### ISSUE THREAD

**Issue by philipgiuliani** [MicheleBertoli/react-gmaps#50]
When i add one marker to the map, its redrawing all markers (blinking).

Code:
```javascript
    render() {
        // i add some markers on some zoomlevels
        const markers = this.state.data.map((e) => {
            return <Marker key={e.id}
                lat={e.location.lat}
                lng={e.location.lng}
                onClick={this.onEventClick.bind(this, e)} />;
        });

        return (
            <Gmaps
                width="100%"
                height="100%"
                lat={51.5258541}
                loadingMessage="Loading..."
                disableDefaultUI={true}
                lng={-0.08040660000006028}
                onMapCreated={this.onMapCreated.bind(this)}
                maxZoom={18}
                styles={mapStyle}
                backgroundColor="#2b2b2b"
                zoom={2}>{markers}</Gmaps>
        );
    }
```

**Comment by MicheleBertoli** [comment]
Hey @philipgiuliani, what version are you using?
The problem you describe should be fixed in #38 (Release [1.4.2](https://github.com/MicheleBertoli/react-gmaps/releases/tag/1.4.2)).

**Comment by philipgiuliani** [comment]
I am using
```
"react": "~> 15.1.0",
"react-dom": "~> 15.1.0",
"react-gmaps": "^1.5.0"
```

Maybe it is because im re-creating all objects on every zoom (content stays the same). I'll try to just append the new ones.
